instrumentation/koala/test/instrumentation/koala/instrumentation_test.rb

```diff
@@ -0,0 +1,52 @@
+# frozen_string_literal: true
+
+# Copyright The OpenTelemetry Authors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+require 'test_helper'
+require 'pry'
+
+require_relative '../../../lib/opentelemetry/instrumentation/koala'
+
+describe OpenTelemetry::Instrumentation::Koala do # rubocop:disable Metrics/BlockLength
+  let(:instrumentation) { OpenTelemetry::Instrumentation::Koala::Instrumentation.instance }
+  let(:exporter) { EXPORTER }
+  let(:span) { exporter.finished_spans.first }
+
+  it 'has #name' do
+    _(instrumentation.name).must_equal 'OpenTelemetry::Instrumentation::Koala'
+  end
+
+  it 'has #version' do
+    _(instrumentation.version).wont_be_nil
+    _(instrumentation.version).wont_be_empty
+  end
+
+  describe '#install' do
+    it 'accepts argument' do
+      instrumentation.install({})
+    end
+  end
+
+  describe 'install' do
+    before do
+      exporter.reset
+      instrumentation.install({})
+    end
+
+    it 'when koala call made' do
+      stub_request(:get, 'https://graph.facebook.com/me?access_token=fake_token').with(
+        headers: {
+          'Accept' => '*/*',
+          'Accept-Encoding' => 'gzip;q=1.0,deflate;q=0.6,identity;q=0.3',
+          'User-Agent' => 'Faraday v1.3.0'
```

Comment:
I think you should be able to just remove the `.with()` portion of this stub to make it a bit more flexible to different versions in CI.
```
  1) Error:
OpenTelemetry::Instrumentation::Koala::install#test_0001_when koala call made:
WebMock::NetConnectNotAllowedError: Real HTTP connections are disabled. Unregistered request: GET https://graph.facebook.com/me?access_token=fake_token with headers {
  'Accept'=>'*/*', 
  'Accept-Encoding'=>'gzip;q=1.0,deflate;q=0.6,identity;q=0.3', 
  'User-Agent'=>'Faraday v1.4.0'
}
```